BryantUniversity/BryantUniversity/scripts/calendar.js

```diff
@@ -1,12 +1,93 @@
-﻿const gebi = (e) => document.getElementById(e);
+﻿(async() => {
 
-let dropDownMenu = gebi("dropdownMenuButton")
-let dropDownOptions = gebi("dropDownOptions")
+    const gebi = (e) => document.getElementById(e);
 
-var selectedYear = dropDownMenu.textContent
+    async function changePeriod(periodId)
+    {
+        try{
+            const response = await fetch ('http://localhost:51934/api/Calendar/' + periodId ,{
+                method: "POST",
+                credentials:"include",
+                header:{
+                    "Context-Type":"application/json"
+                }
+            });
 
+            const data = await response.json();
+            console.log(data);
+            console.log("JSON: " + JSON.stringify(data))
+            return data
+        }
+        catch(error){
+            console.log('ERROR: ' + error);
+        }
+    }
 
-dropDownOptions.onclick = function () {
+    function resetList(){
+        var container = gebi("tableContainer");
```

Comment:
In general, I would avoid using `var` to declare variables in JavaScript. `const` and `let` are less prone to bugs due to variable scope.

Reply:
ill make those changes in another branch